.buildkite/scripts/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TERRAFORM_ROLE="arn:aws:iam::760097843905:role/platform-ci"
+
+# Do this here rather than in Dockerfile so we don't install it on every build/test
+apt install -y awscli zip
```

Comment:
Is this going to install on the host instances? Colour me a bit surprised they're not there already.

Reply:
Inside the docker container - basically had to choose between installing this in the container, or node (etc etc) on the host. Not ideal, but straightforward.